src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -427,4 +427,204 @@ private class ACCT_IndividualAccounts_TEST {
         newAcc = [select id,Name from Account where id = :con.AccountId limit 1];
         system.assertEquals(newAcc.Name, 'Contact_forTests' + ' ' + Label.DefaultAdminName);
     }
+
+    /*********************************************************************************************************
+    * @description Insert a new contact and the default account record type is Household
+    * Make sure the Account's name is set based on account naming setting.
+    */
+    @isTest
+    public static void newContactNewHHAcc() {
+        UTIL_CustomSettingsFacade.getSettingsForTests(new Hierarchy_Settings__c(Account_Processor__c = UTIL_Describe.getHhAccRecTypeID(),
+                Household_Account_Naming_Format__c = '{!FirstName} {!LastName} Household'));
+
+        Contact con = new Contact(
```

Comment:
Based on the discussion, for the new test cases, we need to use UTIL_UnitTestData_TEST to create test Contacts and Accounts now.

Reply:
Yep I have create a workitem for this. Nice catch!